evolver/connection/interface.py

```diff
@@ -48,7 +48,7 @@ def communicate(self, *args, **kwargs):
 
     def is_open(self):
         if hasattr(self.conn, "is_open"):
-            return self.conn.is_open()
+            return self.conn.is_open
```

Comment:
```suggestion
            return self.conn.is_open() if callable(self.conn.is_open) else self.conn.is_open
```
I see your point that this could be a property, but it might not be since this is in the base interface, so we should check if callable etc.

Reply:
I couldn't apply directly for some reason, but made this change, thanks!

Reply:
My bad, I copied and pasted it from a previous post attempt as I also tried to include the ``if`` to suggest using a ``getattr`` rather than ``hasattr`` but GitHub wouldn't let me make a suggestion stradling the deleted line. Perhaps, I added it back to the wrong line. 🤷